client/src/actions/linkedissuesActions.js

```diff
@@ -0,0 +1,45 @@
+import LinkedIssuesRequest from '../requests/linkedissuesRequests';
+import { globalErrorHandler, responseToJson } from '../utils';
+
+export const LinkedIssuesActionsTypes = {
+    LINKEDISSUES_GET_REQUEST: 'LINKEDISSUES_GET_REQUEST',
+    LINKEDISSUES_GET_REQUEST_SUCCESS: 'LINKEDISSUES_GET_REQUEST_SUCCESS',
+    LINKEDISSUES_GET_REQUEST_ERROR: 'LINKEDISSUES_GET_REQUEST_ERROR',
+};
+
+
+
+//getlinkedissues
+export function getBegin() {
```

Comment:
naming: should contain the name of the action ex getBeginLinkedIssues, beginGetLinkedIssued, etc